### PR TITLE
add anchor changes

### PR DESCRIPTION
**Pull request by britt6612**
Add changes from this PR 
Note I didn't not list out all of the `sizes` but can if that might be better
https://github.com/grommet/grommet/pull/6643

### CODE REVIEW

File: src/screens/Anchor.js [hunk 1]
```diff
@@ -283,6 +283,48 @@ const AnchorPage = () => (
           <GenericExtend />
         </Property>
 
+        <Property name="anchor.size.[size].color">
+          <Description>
+            The color of the label text and icon strokes on different Anchor
+            [size] which is the text size of Anchor. sizes.
```
Comment (jcfilben) [inline src/screens/Anchor.js:289]:
```suggestion
            The color of the label text and icon strokes on different Anchor
            sizes. [size] refers to the text size of the Anchor.
```

File: src/screens/Anchor.js [hunk 2]
```diff
@@ -283,6 +283,48 @@ const AnchorPage = () => (
...
+            <Example defaultValue>
+              <Markdown>{`{"dark": "black", "light": "brand"}`}</Markdown>
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="anchor.size.[size].font-weight">
+          <Description>
+            The font weight of the label on different Anchor [size] which is the
+            text size of Anchor.
```
Comment (jcfilben) [inline src/screens/Anchor.js:308]:
```suggestion
            The font weight of the label on different Anchor sizes. [size] refers to the
            text size of Anchor.
```

File: src/screens/Anchor.js [hunk 3]
```diff
@@ -283,6 +283,48 @@ const AnchorPage = () => (
...
+          <PropertyValue type="number">
+            <Example>300</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="anchor.size.[size].textDecoration">
+          <Description>
+            {' '}
+            The text decoration of the label when hovering on Anchor [size]
+            which is the text size of Anchor. Refer to
```
Comment (jcfilben) [inline src/screens/Anchor.js:319]:
```suggestion
            The text decoration of the label when hovering on different Anchor sizes.
            [size] refers to the text size of Anchor. Refer to
```

**Review by jcfilben — APPROVED** [review]
looks good just needs a yarn.lock update

File: src/screens/Anchor.js [hunk 4]
```diff
@@ -283,6 +283,45 @@ const AnchorPage = () => (
           <GenericExtend />
         </Property>
 
+        <Property name="anchor.size.[size].color">
+          <Description>
+            The color of the label text and icon strokes on different Anchor
+            sizes. size refers to the text size of the Anchor.
```
Comment (jcfilben) [inline src/screens/Anchor.js:289]:
```suggestion
            sizes. Size refers to the text size of the Anchor.
```

File: src/screens/Anchor.js [hunk 5]
```diff
@@ -283,6 +283,45 @@ const AnchorPage = () => (
...
+            </Description>
+            <Example defaultValue>
+              <Markdown>{`{"dark": "black", "light": "brand"}`}</Markdown>
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="anchor.size.[size].font-weight">
+          <Description>
+            The font weight of the label on different Anchor sizes. size refers
```
Comment (jcfilben) [inline src/screens/Anchor.js:307]:
```suggestion
            The font weight of the label on different Anchor sizes. Size refers
```

File: src/screens/Anchor.js [hunk 6]
```diff
@@ -283,6 +283,45 @@ const AnchorPage = () => (
...
+          </Description>
+          <PropertyValue type="number">
+            <Example>300</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="anchor.size.[size].textDecoration">
+          <Description>
+            The text decoration of the label when hovering on different Anchor
+            sizes. size refers to the text size of Anchor.
```
Comment (jcfilben) [inline src/screens/Anchor.js:318]:
```suggestion
            sizes. Size refers to the text size of Anchor.
```

File: src/screens/Anchor.js [hunk 7]
```diff
@@ -283,6 +283,45 @@ const AnchorPage = () => (
...
+            <Description>
+              An object with a color for dark and light mode.
+            </Description>
+            <Example defaultValue>
+              <Markdown>{`{"dark": "black", "light": "brand"}`}</Markdown>
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="anchor.size.[size].font-weight">
```
Comment (jcfilben) [inline src/screens/Anchor.js:305]:
```suggestion
        <Property name="anchor.size.[size].fontWeight">
```

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**